stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -403,15 +403,23 @@ ::util::Status GetAllEntries(
   const ::tdi::Device *device = nullptr;
   ::tdi::DevMgr::getInstance().deviceGet(0, &device);
   const auto flags = ::tdi::Flags(0);
-  uint32 entries = 0;
+  tdi_status_t tdi_status;
+  std::unique_ptr<::tdi::Target> dev_tgt;
+  device->createTarget(&dev_tgt);
+  uint32 entries = 0, actual = 0;
```

Comment:
`actual` isn't needed until later in the code. It is better C++ coding practice to define the variable just before it's needed (as the code did before you modified it) instead of moving the definition up here.